operator/webhooks/pod_mutating_webhook.go

```diff
@@ -89,7 +90,7 @@ func (a *PodMutatingWebhook) Handle(ctx context.Context, req admission.Request)
 
 	if !podIsAnnotated {
 		logger.Info("Pod is not annotated, check for parent annotations...")
-		podIsAnnotated, err = a.copyAnnotationsIfParentAnnotated(ctx, &req, pod)
+		podIsAnnotated, _ = a.copyAnnotationsIfParentAnnotated(ctx, &req, pod)
```

Comment:
this could return a  ErrTooLongAnnotations , I think in this case we should have the same as line 86-89 and return an admission error